released/packages/coq-msets-extra/coq-msets-extra.1.2.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+maintainer: "thomas@tuerk-brechen.de"
+homepage: "https://github.com/thtuerk/MSetsExtra"
+dev-repo: "git+https://github.com/thtuerk/MSetsExtra.git"
+bug-reports: "https://github.com/thtuerk/MSetsExtra/issues"
+authors: ["FireEye Formal Methods Team <formal-methods@fireeye.com>" "Thomas Tuerk <thomas@tuerk-brechen.de>"]
+license: "LGPL 2.1"
+build: [
+  ["./configure.sh"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/MSetsExtra"]
+depends: [
+  "ocaml"
+  "coq" {>= "8.9" & < "8.10~"}
+  "coq-mathcomp-ssreflect" {>= "1.6"}
+]
+tags: [ "keyword:finite sets"
+        "keyword:fold with abort"
+        "keyword:extracting efficient code"
+        "keyword:data structures"
+        "category:Computer Science/Data Types and Data Structures"
+        "category:Miscellaneous/Extracted Programs/Data structures"
```

Comment:
```suggestion
        "category:Miscellaneous/Extracted Programs/Data structures"
        "logpath:MSetsExtra"
```
Adding logical path (user-contrib directory name) here makes the package more searchable.